docs/source/support/faq.rst

```diff
@@ -0,0 +1,168 @@
+
+
+What is the network name for Incentivized Testnet?
+--------------------------------------------------
+
+The network name is ``cassandra``.
+
+
+How long is a period?
+---------------------
+
+1 period equals 24 hours. Periods begin at midnight UTC.
+
+
+What's the difference between staker address and worker address? Can they be the same?
+--------------------------------------------------------------------------------------
+
+Technically, yes, but it is not recommended. The accounts have different security considerations - the staker address
+can be a high-value hardware wallet (with NU and ETH) that performs stake management while the worker
+address is low-value and needs to remain unlocked while running (software wallet with ETH) since it
+is used by an Ursula node.
+
+You should stake with one address and set the worker to be a different address. Subsequently, you can bond
+the worker address to the stake.
+
+
+Is there a guide for Windows?
+-----------------------------
+
+Our guide is intended for Linux - we do not officially support Windows.
+
+
+Where is my Ursula config path?
+-------------------------------
+
+On Linux - ``$HOME/.local/share/nucypher/ursula.json``
+
+
+What is the difference between Standard Installation and Development Installation?
+----------------------------------------------------------------------------------
+
+The Development Installation is only needed for developing with ``nucypher``. You don't need to use
+it unless you plan to make changes to the codebase. If you are simply staking/running a node, you
+only need the Standard Installation
+
+
+How do I know that my node is set up correctly?
+-----------------------------------------------
+
+This is **ONLY** a heuristic to ensure that your node is running correctly, it doesn't guarantee your node is setup correctly: 
+
+    #. Ensure that your Ursula node is up and running (logs/terminal):
+
+       .. code::
+
+            Starting Ursula on xxx.xxx.xxx.xxx:9151
+            Connecting to cassandra
+            Working ~ Keep Ursula Online!
+
+    #. Ensure that your node uses the correct IP address and can be accessed via port 9151 from an outside
+       connection eg. cell phone, other computer etc. by navigating to: ``https://<node_ip>:9151/status``
+
+    #. Ensure that your worker is bonded with your staker - ``nucypher stake list`` and check that
+       *Worker* is set correctly i.e. not ``0x0000``.
+
+    #. Ensure that your node is listed on the `Status Monitor Page <https://status.nucypher.network>`_ (this can take a few mins).
+
+       If your node is on the status monitor page but:
+
+        a. Does not have a green dot, you should ensure that your Ursula node can confirm activity (hint: does your worker address have ETH to pay gas?).
+           Try locally running ``nucypher ursula confirm-activity``
```

Comment:
Should we be emphasizing this command?

Reply:
No, we shouldn't as it will be discontinued any time soon.

Reply:
Would you rather this entire point (a) be removed then?